documentation/asciidoc/accessories/camera/libcamera_apps_getting_started.adoc

```diff
@@ -59,6 +59,6 @@ To override the automatic camera detection, _Bullseye_ users will also need to d
 
 `camera_auto_detect=1`
 
-if present in the `config.txt` file. If you need to do edit this file then your Raspberry Pi will need to be rebooted. 
+If present in the `config.txt` file. If you need to do edit this file then your Raspberry Pi will need to be rebooted. 
```

Comment:
So if you look in context,

<img width="1037" alt="Screenshot 2022-02-28 at 15 16 31" src="https://user-images.githubusercontent.com/53625/156008196-3e36d85f-1493-4080-9d47-e05fedac6be1.png">

this "if" is intended to be the continuation of the sentence started before the code block. What should probably be done is close up the paragraph so that `camera_auto_detect=1` is inline in the sentence rather than separated out.

Reply:
Fixed!

Reply:
Thanks